.github/workflows/ci.yml

```diff
@@ -98,3 +98,51 @@ jobs:
           - { EXPECTED_FAILURES: "or1k-cy or1k-cmov", EXTRA_CORE_ARGS: "--feature_cmov NONE" }
           - { EXPECTED_FAILURES: "or1k-cy or1k-ext", EXTRA_CORE_ARGS: "--feature_ext NONE" }
 
+  formal_tests:
+    runs-on: ubuntu-18.04
+    steps:
+      #Checkout Repository
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      #Install Yosys-Formal tools
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install g++ gperf build-essential bison flex libreadline-dev gawk tcl-dev libffi-dev git graphviz xdot pkg-config python python3 libboost-system-dev libboost-python-dev libboost-filesystem-dev zlib1g-dev
+
+      - name: Install clang
+        run: |
+          sudo apt-get -y install clang
+
+      - name: Install yosys
+        run: |
+          mkdir -p formal_tools
+          cd formal_tools
+          git clone --depth 1 https://github.com/YosysHQ/yosys.git yosys
```

Comment:
No version... you said it breaks something.  OK, we can do something about it later.